### PR TITLE
Beta distribution natural parameters

### PR DESCRIPTION
**Pull request by Nimrais**
The purpose of this pull request is to make the functionality of the CVI **to run** with the Beta distribution. The following changes have been made: `BetaNaturalParameters` is implemented (with tests): the logpdf, the lognormalizer, constructors.

Also, a basic CVI **run** test is added: it checks only that CVI is running with Beta, the actual prod quality is not checked.

### CODE REVIEW

File: test/distributions/test_beta.jl [hunk 1]
```diff
@@ -37,6 +37,38 @@ import ReactiveMP: mirrorlog
...
+
+        # @testset "lognormalizer" begin
+        #     @test lognormalizer(UnivariateNormalNaturalParameters(1, -2)) ≈ -(log(2) - 1 / 8)
+        # end
+
+        # @testset "logpdf" begin
+        #     for i in 1:10
+        #         @test logpdf(UnivariateNormalNaturalParameters(i, -i), 0) ≈ logpdf(NormalWeightedMeanPrecision(i, 2 * i), 0)
+        #     end
+        # end
```
Comment (albertpod) [inline test/distributions/test_beta.jl:61]: Why is it commented?

Comment (Nimrais) [reply test/distributions/test_beta.jl:61]: Because it's wrong. It should be implemented.

Comment (Nimrais) [reply test/distributions/test_beta.jl:61]: This tests are just copy paste from the Gaussians to clearly see for what I need to write tests.

Comment (Nimrais) [reply test/distributions/test_beta.jl]: [done](https://github.com/biaslab/ReactiveMP.jl/pull/274/commits/f3eba15ac163e71e9270abd3ea5291676a104bb3)

**Review by ismailsenoz — APPROVED** [review]

**Review by bartvanerp — APPROVED** [review]

**Comment by bvdmitri** [comment]
@Nimrais why should I understand? I have no idea what is wrong with Zygote on nightly :)

**Comment by bvdmitri** [comment]
It may be an upstream problem in Turing, we can wait some time and re-run the test-suit to see if it resolves the issue. If not, we should figure it out ourselves

**Comment by Nimrais** [comment]
> @Nimrais why should I understand? I have no idea what is wrong with Zygote on nightly :)

I mean we can not decide to merge it or not without you if it fails on nightly.

**Review by bvdmitri — APPROVED** [review]

**Pull request merged**